Lib/test/test_binhex.py

```diff
@@ -3,10 +3,12 @@
    Uses the mechanism of the python binhex module
    Based on an original test by Roger E. Masse.
 """
-import binhex
 import unittest
 from test import support
 
+with support.check_warnings(('', DeprecationWarning)):
+    import binhex
```

Comment:
Is `binhex` imported in `test___all__` or other tests?

Reply:
The full test suite pass with -Werror, except of test_signal which is unrelated to this PR: https://bugs.python.org/issue39424